src/main/java/org/c4sg/service/FeedService.java

```diff
@@ -0,0 +1,66 @@
+package org.c4sg.service;
+
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+import org.c4sg.dto.OrganizationDTO;
+import org.c4sg.dto.afg.FeedDateTimeDurationDTO;
+import org.c4sg.dto.afg.FeedInfoDTO;
+import org.c4sg.dto.afg.FeedLocationDTO;
+import org.c4sg.dto.afg.FeedSponsoringOrganizationDTO;
+import org.springframework.stereotype.Service;
+
+@Service
+public class FeedService {
```

Comment:
I'm getting a lot of conflicts as well in 3 classes: OrganizationDTO, WebSecurityConfig and JwUtil.